src/services/dexes/sovryn/SovrynPriceFetcher.ts

```diff
@@ -0,0 +1,20 @@
+import {PricesResponse, PairRequest, SovrynHelperBase} from '../../../blockchains/evm/contracts/SovrynHelper.js';
+import {FeedFetcherInterface} from 'src/types/fetchers.js';
+
+export class SovrynPriceFetcher implements FeedFetcherInterface {
```

Comment:
let's add a comment with example of price configuration for this fetcher

Reply:
autor of comment should resole conversation :)
sorry, I was not precise: I ment comment about how to configure yaml eg https://github.com/umbrella-network/pegasus/blob/develop/src/services/fetchers/EvmTWAPGasPriceFetcher.ts#L6

Reply:
That is true, sorry for resolving the conversation. Thanks for the extra details :)